fhir-install/pom.xml

```diff
@@ -356,5 +306,129 @@
+                                            <groupId>${project.groupId}</groupId>
+                                            <artifactId>fhir-ig-us-core</artifactId>
+                                            <version>${project.version}</version>
+                                            <overWrite>false</overWrite>
+                                        </artifactItem>
+                                        <artifactItem>
+                                            <groupId>${project.groupId}</groupId>
+                                            <artifactId>fhir-ig-carin-bb</artifactId>
+                                            <version>${project.version}</version>
+                                            <overWrite>false</overWrite>
+                                        </artifactItem>
+                                        <artifactItem>
+                                            <groupId>${project.groupId}</groupId>
+                                            <artifactId>fhir-ig-mcode</artifactId>
+                                            <version>${project.version}</version>
+                                            <overWrite>false</overWrite>
+                                        </artifactItem>
+                                        <artifactItem>
+                                            <groupId>${project.groupId}</groupId>
+                                            <artifactId>fhir-ig-davinci-pdex-plan-net</artifactId>
+                                            <version>${project.version}</version>
+                                            <overWrite>false</overWrite>
+                                        </artifactItem>
+                                        <artifactItem>
+                                            <groupId>${project.groupId}</groupId>
+                                            <artifactId>fhir-ig-davinci-pdex-formulary</artifactId>
+                                            <version>${project.version}</version>
+                                            <overWrite>false</overWrite>
+                                        </artifactItem>
+                                    </artifactItems>
+                                    <overWriteReleases>false</overWriteReleases>
+                                    <overWriteSnapshots>true</overWriteSnapshots>
+                                    <outputDirectory>${project.build.directory}/fhir-igs</outputDirectory>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+
+        <profile>
+            <id>fhir-install-distribution</id>
+            <activation>
+                <property>
+                    <name>!skipFhirInstallDistribution</name>
+                </property>
+            </activation>
```

Comment:
interesting approach... use activatedByDefault?

Reply:
With activatedByDefault by default, if another profile is used, then this wouldn't run.  I think I want this one to always run.

Reply:
This doesn't make sense in a profile then

Reply:
I was able to leave this within a regular build tag, instead of within a profile build tag.  When I had it copying the IG jars too, I think I had needed to make sure of the profile order, but now I can just have it the way it was.